.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 *.log
+.idea/
```

Comment:
this is not related to the project, as I've learnt recently
I suggest you to check this out: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore